v2/config/rbac/classic/role.yaml

```diff
@@ -245,12 +306,10 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: reporter
-  namespace: openshift-config
+  namespace: system
```

Comment:
Bit of an explanation here. Originally we needed a role in the openshift-config NS to access the secrets there for uploading. Since we've removed that functionality this is fine.